catalyst-gateway/bin/src/service/api/test_endpoints/mod.rs

```diff
@@ -108,8 +108,8 @@ impl TestApi {
         /// The action just needs to be any valid UUID.
         ///
         /// # Make sure its a UUID
-        _action: Path<Option<String>>,
+        action: Path<String>,
```

Comment:
Maybe it is good time just to remove these test endpoints ?
They have added as a reference how to work with the `poem-openapi` lib.
@stevenj

Reply:
Steven told me about this, maybe will leave it there but just comment it out